lib/build-metadata.ts

```diff
@@ -202,9 +202,9 @@ export class BuildMetadata {
 		path: string,
 	): { relativePath: string; metadataDirectory: string } | undefined {
 		for (const metadataDirectory of this.metadataDirectories) {
-			if (PathUtils.contains(metadataDirectory, path)) {
+			if (PathUtils.posixContains(metadataDirectory, path)) {
```

Comment:
Can you please change `PathUtils` so that `contains` is part of its `posix` namespace? All the other commands are `PathUtils.posix.<command>` but this is on `PathUtils` which is inconsistent. It doesn't have to be on this PR if you don't feel like though.

Reply:
Whether or not this is a good reason, the reason is that `PathUtils.posix.*` is a plain re-export of the standard Node `path.posix.*` (built-in lib), whereas `PathUtils.contains` and `PathUtils.posixContains` are our own functions. In other words, there is currently consistency in that neither `path.contains` nor `path.posix.contains` exist. :-)
```
# resin-multibuild/lib/path-utils.ts

import { posix } from 'path';
export * from 'path';

export const contains = ... ;
export const posixContains = ... ;
```

Reply:
So I think I'll leave the `posixContains` → `posix.contains` change to a future PR.